.github/workflows/github-pages.yml

```diff
@@ -24,27 +24,33 @@ jobs:
       - name: Git Checkout
         uses: actions/checkout@v3
 
+      - name: Install PNPM
+        uses: pnpm/action-setup@v2
+        with:
+          version: latest
+
```

Comment:
```suggestion
```

Reply:
Hey @aduh95 any instructions for using Corepack?

Reply:
Because if I don't provide the install action for PNPM, then `actions/setup-node@v3` fails

Reply:
AFAIK there's shouldn't need any install step as it is installed by default in GHA runner along `node`.